dll/shellext/fontext/CFontExt.cpp

```diff
@@ -151,41 +163,28 @@ STDMETHODIMP CFontExt::GetDetailsOf(PCUITEMID_CHILD pidl, UINT iColumn, SHELLDET
     }
 
     // If we got here, we are in details view!
-    // Let's see if we got info about this file that we can re-use
-    if (m_LastDetailsFontName != fontEntry->Name)
+    auto info = g_FontCache->Find(fontEntry);
```

Comment:
Didn't know it was so complicated to know the type of "info", so that you need to let the compiler determine it...

Reply:
yes, I almost sprained my fingers while typing, so used auto.